lua/efmls-configs/formatters/deno-fmt.lua

```diff
@@ -0,0 +1,15 @@
+-- Metadata
+-- languages: javascript,typescript
```

Comment:
I see that it can format other files too, it'd be nice to add them here as well like `javascriptreact` (jsx), `typescriptreaact` (tsx), etc. for documentation.

Taking from https://docs.deno.com/runtime/manual/tools/formatter